lib/src/authorization.rs

```diff
@@ -120,3 +120,7 @@ pub enum EncodingError {
     #[error(transparent)]
     Base64(#[from] base64::DecodeError),
 }
+
+pub enum Query {
```

Comment:
Should this be called `CapabilitiesQuery` or will it be used for all services?

Reply:
that's sensible, fixed